File: course.py
from prodi import Prodi
# Kelas Course sebagai child dari kelas Prodi
class Course(Prodi) :
    #access modifier private
    __nama_matakuliah ="";

    # constructor
    def __init__(self, nama_matakuliah = ""):
        self.__nama_matakuliah = nama_matakuliah

    # getter and setter
    def get_nama_matakuliah(self):
        return self.__nama_matakuliah
    
    def set_nama_matakuliah(self, nama_matakuliah):
        self.__nama_matakuliah = nama_matakuliah

File: cpp/course.cpp
//import library
#include <bits/stdc++.h>
#include "prodi.cpp"
//using standard namespace
using namespace std;
//Human sebagai anak yang diwariskan dari Mahasiswa
class Course : public Prodi
{
    //access modifier private
    private:
        string nama_matakuliah;

    //access modifier publik    
    public:
        //constructor
        Course()
        {
            this->nama_matakuliah = " ";
        }

        //penggunaan getter dan setter
        string get_nama_matakuliah()
        {
            return this->nama_matakuliah;
        }

        void set_nama_matakuliah(string nama_matakuliah)
        {
            this->nama_matakuliah = nama_matakuliah;
        }
        

        //destructor
        ~Course()
        {

        }
};